src/XeroPHP/Remote/Query.php

```diff
@@ -42,22 +60,63 @@ public function from($class) {
     }
 
     /**
+     * @param string $where
+     * @param mixed|null $value
      * @return $this
      */
-    public function where() {
-        $args = func_get_args();
-
-        if(func_num_args() === 2) {
-            $this->where[] = sprintf('%s=="%s"', $args[0], $args[1]);
+    public function where($where, $value = null) {
+        if ($value !== null) {
+            $parameterName = $this->addInternalParameter($value);
+            $this->where[] = sprintf('%s==%s', $where, ":$parameterName");
```

Comment:
That seems now to be overcomplicated!  I was just meaning to use something that'd be unique instead of that sha1.